pom.xml

```
@@ -25,11 +25,11 @@
     </scm>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
```

```suggestion
        <java.version>17</java.version>
```

Since this is running inside the broker.